packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -218,8 +219,48 @@ export const GCNodeType = {
  */
 export type GCNodeType = (typeof GCNodeType)[keyof typeof GCNodeType];
```

Comment:
FYI this type is also equivalent to `string` 🤷‍♂️

Reply:
I took this from Tony's suggestion of not using an enum because it has certain issues. This is basically supposed to be equivalent to enum. The type should be the types of the const above.

Reply:
Yeah I was surprised.  Not sure if something changed with TS.  You can fix it by adding `as const` after each string value in the cost GCNodeType object.  You can verify the problem/fix by merely hovering over `GCNodeType` (the type not the const)

Reply:
Actually you can just do `as const` after the whole thing, don't need it on each line